var/spack/repos/builtin/packages/py-wxpython/package.py

```diff
@@ -27,3 +29,8 @@ class PyWxpython(PythonPackage):
     depends_on("py-numpy", type=("build", "run"))
     depends_on("pil", type=("build", "run"))
     depends_on("py-six", type=("build", "run"))
+
+    def setup_build_environment(self, env):
+        # By default wxWdigets is built as well instead of using spack provided version,
+        # this tells it to just build the python extensions
+        env.set("WXPYTHON_BUILD_ARGS", "build_py --use_syswx")
```

Comment:
I wonder if this option existed at the time the first version in spack was added (since it seems someone, *cough* @adamjstewart *cough*, was able to build wxpython at that time without the pkgconfig and gtkplus dependencies). Did you check that providing the `--use_syswx` flag does not lead to failures building the 4.0.6 version if it did not suppport that flag?

Reply:
I can give try to test this out soon.

Reply:
The 4.0.6 version does understand the `--use_syswx` option, so I assume it was using the bundled wxwidgets, and the build was succeeding somehow with out `pkg_config` included as a build dep.